### PR TITLE
Update Cloud SQL for SQL Server sample app w/ Kubernetes Engine

### LINKED ISSUE #7413: Update Cloud SQL : SQL Server sample app to enable Kubernetes Engine deployment
## Describe the issue

The [Cloud SQL : SQL Server](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/cloud-sql/sql-server/sqlalchemy) sample app needs to be updated to support deployment of the app to Kubernetes Engine using Private IP,
similar to the existing Kubernetes Engine deployment support included in the [Cloud SQL : MySQL](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/cloud-sql/mysql/sqlalchemy) and[ Cloud SQL : Postgres](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/cloud-sql/postgres/sqlalchemy) sample apps.

### PR DESCRIPTION
**Pull request by jsimonweb**
## Description

Fixes #7413

## Checklist

- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: cloud-sql/sql-server/sqlalchemy/Dockerfile [hunk 1]
```diff
@@ -45,7 +45,8 @@ COPY . ./
 COPY ./certs /app/certs
 
 # Use server certificate for encrypted connection.
-COPY ./certs/server-ca.pem /usr/local/share/ca-certificates/server-ca.crt
+COPY Dockerfile ./certs/server-ca.pem* /usr/local/share/ca-certificates/
+RUN mv /usr/local/share/ca-certificates/server-ca.pem /usr/local/share/ca-certificates/server-ca.crt; exit 0
```
Comment (enocom) [inline cloud-sql/sql-server/sqlalchemy/Dockerfile:49]: Why is this rename here? Does using server-ca.pem break somewhere?

**Review by enocom — APPROVED** [review]
LGTM. I'd like @jackwotherspoon to take a look too.

File: cloud-sql/sql-server/sqlalchemy/Dockerfile (hunk 1 above)
Comment (jsimonweb) [reply cloud-sql/sql-server/sqlalchemy/Dockerfile:49]: This addresses the case where the ` server-ca.pem` file is not provided.
The first line conditionally copies the ` server-ca.pem` file and the second line renames the copied file if it exists.

Comment (enocom) [reply cloud-sql/sql-server/sqlalchemy/Dockerfile:49]: In that case, a comment explaining why this is happening be useful.

Comment (jsimonweb) [reply cloud-sql/sql-server/sqlalchemy/Dockerfile:51]: Done.

**Review by jackwotherspoon — APPROVED** [review]

**Review by busunkim96 — APPROVED** [review]

**Pull request merged**